.circleci/soltest.sh

```diff
@@ -55,7 +55,12 @@ get_logfile_basename() {
 }
 
 BOOST_TEST_ARGS="--color_output=no --show_progress=yes --logger=JUNIT,error,test_results/`get_logfile_basename`.xml"
-SOLTEST_ARGS="--evm-version=$EVM --evmonepath /usr/lib/libevmone.so $SOLTEST_FLAGS"
+if [ "$(uname)" == "Darwin" ]
+then
+  SOLTEST_ARGS="--evm-version=$EVM --evmonepath /Users/distiller/project/lib/libevmone.dylib $SOLTEST_FLAGS"
```

Comment:
We should instead of this be able to remove ``--evmonepath /usr/lib/libevmone.so`` entirely (``/usr/lib`` is in the default search path anyways) and if you then move ``libevmone.dylib`` into ``deps/`` or ``deps/lib`` in the repo sources instead (both should work), then we don't need any of this...

Reply:
(if you're interested see ``EVMOneEnvOrDefaultPath`` in ``test/Common.cpp`` for the search logic)

Reply:
ok that means we could totally remove `--evmonepath`?

Reply:
because `libevmone.dylib` is either in `/usr/lib` or `/usr/local/lib`.. and both where defined in `searchPath` inside `test/Common.cpp`